### PR TITLE
Blog post to announce obs version 2.8

### PR DESCRIPTION
**Pull request by evanrolfe**
https://trello.com/c/NVHM3zTS/337-5-p3-release-announcement-2-8

### CODE REVIEW

**Comment by bgeuken** [comment]
Looks great! 🎉

File: _posts/releases/2017-03-28-version-2.8.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+For a more detailed look at whats new in 2.8 please check out the [release notes](https://github.com/openSUSE/open-build-service/blob/2.8/ReleaseNotes-2.8).
+## Try OBS 2.8
+The latest OBS version version is always deployed to our reference server. You can try it out there.
+
+The reference server [http://build.opensuse.org](http://build.opensuse.org) is available for all open source developers to build packages for the most popular distributions including openSUSE, Debian, Fedora, Ubuntu, Arch, Red Hat Enterprise Linux and SUSE Linux Enterprise. It is also used to build, release and maintain the [openSUSE](http://www.opensuse.org/) distribution.
+## Install 2.8
+Please read our setup instructions or better yet, use our appliance.
```
Comment (Ana06) [inline _posts/releases/2017-03-28-version-2.8.md]: maybe linking the instructions and the appliance?

**Comment by Ana06** [comment]
@evanrolfe can you please change the date to tomorrow (`2017-04-05`)? It will be ready tomorrow, so if you change the date tomorrow when it is finished we can just merge this. :wink:

File: _posts/releases/2017-04-07-version-2.8.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+For a more detailed look at whats new in 2.8 please check out the [release notes](https://github.com/openSUSE/open-build-service/blob/2.8/ReleaseNotes-2.8).
+## Try OBS 2.8
+The latest OBS version version is always deployed to our reference server. You can try it out there.
+
+The reference server [http://build.opensuse.org](http://build.opensuse.org) is available for all open source developers to build packages for the most popular distributions including openSUSE, Debian, Fedora, Ubuntu, Arch, Red Hat Enterprise Linux and SUSE Linux Enterprise. It is also used to build, release and maintain the [openSUSE](http://www.opensuse.org/) distribution.
```
Comment (hennevogel) [inline _posts/releases/2017-04-07-version-2.8.md]: There is an include file for this:

```
{% include reference_server.inc %}
```

File: _posts/releases/2017-04-07-version-2.8.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+## Install 2.8
+Please read our setup instructions or better yet, use our appliance.
```
Comment (hennevogel) [inline _posts/releases/2017-04-07-version-2.8.md]: Please tell people where they are. 
```
Please read our [setup instructions](https://github.com/openSUSE/open-build-service/tree/2.8#installation)
or better yet, use our [appliance](http://openbuildservice.org/download/)
```

File: _posts/releases/2017-04-07-version-2.8.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Upgrade to 2.8
+If you already have a running OBS installation and want to update it, please read also the [README.UPDATERS](https://github.com/openSUSE/open-build-service/blob/2.8/dist/README.UPDATERS) file about the necessary steps.
+
+OBS Appliance users who have setup their LVM according to our [documentation](http://openbuildservice.org/download/#appliance_config) can just replace their appliance image without data loss. The migration will happen automatically.
+
+## About the Open Build Service
```
Comment (hennevogel) [inline _posts/releases/2017-04-07-version-2.8.md]: There is an include file for this: 
```
{% include about.inc %}
```

**Comment by evanrolfe** [comment]
@hennevogel done.

**Pull request merged**